doc/installation.md

```diff
@@ -174,6 +178,10 @@ packages=com.navercorp
 
 - Setting up the agent via `-javaagent:path/to/scavenger-agent.jar`
 - Specify the config file via `-Dscavenger.configuration=path/to/scavenger.conf`
+- To change the default value of 10MB for the max message size of gRPC in agent, change the setting below.
+- `-Dscavenger.max-message-size`
+    - Supports `B`, `KB`, `MB`, `GB`, and `TB` units.
+> **We recommend tuning your codeBase configuration before changing `scavenger.max-message-size`.**
```

Comment:
```suggestion
> **We recommend tuning your codeBase configuration for minimizing tracing ranges before changing `scavenger.max-message-size`.**
```

Reply:
fixed it in 755bb3d91980ac24c9572a060398d1c60a1b5bd7